### PR TITLE
Add support for ETCD_ENDPOINTS

### PR DESCRIPTION
**Pull request by adidenko**
Allow to specify ETCD_ENDPOINTS instead of ETCD_AUTHORITY.

### CODE REVIEW

**Comment by caseydavenport** [comment]
@adidenko thanks!

@robbrockbank I think we want this for v2.0 - didn't realize this was missing!

**Comment by adidenko** [comment]
Hey guys! Any updates on this? I'd like to get rid of custom (forked) image in Kargo for calico-routereflector support and use upstream one. But it needs this fix and some tag assigned to the image to avoid using `latest` :) Thanks!

File: build_routereflector/README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Run the following command to start the Route Reflector container image.
 docker run -privileged -net=host -d                                \ 
            -e IP=<IPv4_RR>                                         \
            [-e IP6=<IPv6_RR>]                                      \
-           -e ETCD_AUTHORITY=<ETCD_IP:PORT>                        \
+           -e ETCD_ENDPOINTS=<http://NODE:PORT>                    \
```
Comment (robbrockbank) [inline build_routereflector/README.md]: minor nit:  wording here changed to NODE - can you change the one below to NODE too?  Then LGTM.

Comment (adidenko) [inline build_routereflector/README.md]: I think I rather change NODE to ETCD_IP since ETCD_IP is used in other parts of README as well.

**Comment by robbrockbank** [comment]
Awesome!  LGTM.

**Comment by robbrockbank** [comment]
We'll look into doing a new RR release as part of our next Calico release.

**Pull request merged**